src/Player.ts

```diff
@@ -16,8 +16,17 @@ import { WSStreamData } from './metadata/WSStreamData';
 import { MBRAbstract, MBRParams } from './mbr/MBRAbstract';
 import { MBRLinear } from './mbr/MBRLinear';
 
-const RECONNECTION_TIMEOUT: number = 1000;
+const RECONNECTION_TIMEOUT: number = 2000;
```

Comment:
I see that this change should be inside a new commit with an explanation

Reply:
and with a BREAKING CHANGE what I was forgotten